tasks/transform_conformance/tests/babel-plugin-transform-typescript/test/fixtures/use-define-for-class-fields-without-class-properties/input.ts

```diff
@@ -0,0 +1,60 @@
+class Cls {
+	x: number;
+	y = 1;
+	@dce
+	z;
+
+	[x]: number;
+	[y] = 1;
+	@dce
+	[z]: number;
+}
+
+class ClsWithConstructor extends Cls {
+	constructor() {
+		console.log("constructor");
+		super();
+	}
```

Comment:
```suggestion
	constructor() {
		console.log("constructor1");
		super();
		console.log("constructor2");
	}
```
I think it'd be nice to change the test case like this to ensure the assignments are injected right after the `super` call.
[TS playground](https://www.typescriptlang.org/play/?target=9&useDefineForClassFields=false#code/MYGwhgzhAEBCkFNoG8C+AodpIwGIHt9oEAPAFwQDsATGeCJZdaaMaAXmgEZnph9KEMgCcArsDL5hACgCUKXi36D8IBADoQ+AObSA5MqFiJXAIR7ZAbkXQIogA4IZVm4dUatugwKPiyAJnMXFgwWACMObnQMIA)

Reply:
Good suggestion!